tests/vcs/git/test_backend.py

```diff
@@ -55,15 +93,27 @@ def test_annotated_tag(tag: str | bytes) -> None:
     assert tag == b"my-tag^{}"
 
 
-def test_get_remote_url() -> None:
-    repo = MagicMock(spec=Repo)
-    repo.get_config.return_value.get.return_value = (
-        b"https://github.com/python-poetry/poetry.git"
-    )
+def test_get_remote_url(git_mock: Repo) -> None:
+    repo = git_mock
 
     assert Git.get_remote_url(repo) == "https://github.com/python-poetry/poetry.git"
 
 
+def test_get_revision(git_mock: Repo) -> None:
+    repo = git_mock
+    assert Git.get_revision(repo) == MOCK_DEFAULT_GIT_REVISION
```

Comment:
```suggestion
    assert Git.get_revision(git_mock) == MOCK_DEFAULT_GIT_REVISION
```

I do not think the additional assignment gives much value. Maybe, we should call the mock `repo_mock` in the first place?